### ISSUE TITLE
Lookup View is an empty view in both VS Code and Codium when on Linux

### ISSUE THREAD

**Issue by ScriptAutomate** [dendronhq/dendron#2464]
## Please select the area your bug applies to. (Multiple selections are Possible. You can leave blank if you're not sure.)

- [ ] Workspace.  VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup.  Dendron's Lookup Command
- [x] Views.  Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema.  Dendron Schemas
- [ ] Pod.  Data import from / export to Dendron
- [ ] Publish.  External Site Publish
- [ ] Markdown.  Markdown features, such as syntax support and features

## To Reproduce

1. Be on a Linux OS, such as Pop!_OS, and inside a workspace within either VS Code or Codium
2. Open Lookup
3. Open the Lookup View while Lookup is open
4. See nothing populating

## Expected behavior

[Lookup View](https://wiki.dendron.so/notes/f7ebd4aa-8ba7-4bc5-bd00-a1efc5315f07/#lookup-view) should contain something.

## Screenshots

![image](https://user-images.githubusercontent.com/5951023/155034939-3e88585e-964b-485b-b49f-52077837521b.png)

## Desktop (please complete the following information)

- OS: Pop!_OS 21.10
- Versions (all that apply to bug):
  - Dendron Extension: `0.82.0`
  - VS Code version: `1.64.2`
  - VSCodium version: `1.64.2`

**Comment by SeriousBug** [comment]
@hikchoi Adding since I believe you added the lookup view and might know more. Let me know if you are having trouble diagnosing or reproducing if this is happening on linux only.

**Comment by hikchoi** [comment]
@SeriousBug Here is the task that has notes on the attempts to diagnose this issue: `[[Investigate Lookup View Bundling Issue|dendron://private/task.2022.01.21.investigate-lookup-view-bundling-issue]]`

Working on a VM was slowing me down a bit. If you can box out a short session to look into this that would be great, since you are on linux. Otherwise I plan to pick this up again during on-call. I should get a proper linux setup going for debugging😅

**Comment by SeriousBug** [comment]
I have quite a bit on my plate so I don't think I can get to this before your oncall, but if I find some free time then I'll leave a message here and pick up the issue.

**Comment by SeriousBug** [comment]
I'm also having an error with lookup view, apart from not showing up, it's throwing an error. The error message is:

```
error while running command: View provider for 'dendron.lookup-view' already registered
```

The error logs:

```
{"level":50,"time":1649038527077,"pid":2620840,"hostname":"eclipse","name":"dendron","ctx":"dendron.lookupSchema:run","error":{"name":"DendronError","status":"unknown","payload":"{}","innerError":{"vslsStack":[{},{},{},{},{},{},{},{},{},{},{}]}}}

``` 

No other errors in the logs or anywhere else.

**Comment by hikchoi** [comment]
> I'm also having an error with lookup view, apart from not showing up, it's throwing an error. The error message is:
> 
> ```
> error while running command: View provider for 'dendron.lookup-view' already registered
> ```
> 
> The error logs:
> 
> ```
> {"level":50,"time":1649038527077,"pid":2620840,"hostname":"eclipse","name":"dendron","ctx":"dendron.lookupSchema:run","error":{"name":"DendronError","status":"unknown","payload":"{}","innerError":{"vslsStack":[{},{},{},{},{},{},{},{},{},{},{}]}}}
> ```
> 
> No other errors in the logs or anywhere else.

This looks like a regression from the recent lookup related circ dependency fixes. pinging @jonathanyeung for visibility.

**Comment by SeriousBug** [comment]
This seems to be fixed for me now (Nightly 0.92.2)!

**Comment by SeriousBug** [comment]
I'm closing this issue since it seems to be resolved now, if anyone on Linux is still having trouble please feel free to reopen.
